browser/brave_content_browser_client_browsertest.cc

```diff
@@ -730,3 +708,21 @@ IN_PROC_BROWSER_TEST_F(BraveContentBrowserClientReferrerTest,
                               &referrer);
   EXPECT_EQ(referrer->url, kDocumentUrl);
 }
+
+// Confirm only expected extension has been installed.
+IN_PROC_BROWSER_TEST_F(BraveContentBrowserClientTest, CheckExpectedExtensions) {
+  // Info: This checkup will not cover on-demand component extension
+  // installation.
+  std::set<std::string> expected_extensions = {
+      brave_extension_id,
+      extensions::kWebStoreAppId,
+      extension_misc::kPdfExtensionId,
+  };
+
+  extensions::ExtensionRegistry* registry =
+      extensions::ExtensionRegistry::Get(browser()->profile());
+  std::set<std::string> installed_extensions =
+      registry->GenerateInstalledExtensionsSet().GetIDs();
+
+  EXPECT_EQ(expected_extensions, installed_extensions);
```

Comment:
I wonder will this actually output the content of both sets if they're not the same?

I think this should output the actual content: `EXPECT_THAT(installed_extensions, testing::ElementsAreArray(expected_extensions));`. Can you try that?

Reply:
Actually EXPECT_EQ will give the output of both Set if its not equal, I checked that before itself.